doc/user/layouts/partials/create-source/connector/file/details.html

```diff
@@ -5,3 +5,9 @@
 ```sql
 CREATE SOURCE server_source FROM FILE '/Users/sean/server.log'...
 ```
+
+#### Compression
+
+* Omitting `COMPRESSION` is equivalent to `COMPRESSION NONE`.
```

Comment:
Very minor, but it looks like you have to go through CI again––we use `-` for `ul` `li`s instead of `*` throughout the docs; nbd but have a slight preference that we keep those consistent for grepability

Reply:
Makes sense, fixed